core/variant/callable.h

```diff
@@ -153,6 +154,7 @@ class CallableCustom {
 	virtual void call(const Variant **p_arguments, int p_argcount, Variant &r_return_value, Callable::CallError &r_call_error) const = 0;
 	virtual Error rpc(int p_peer_id, const Variant **p_arguments, int p_argcount, Callable::CallError &r_call_error) const;
 	virtual const Callable *get_base_comparator() const;
+	virtual int get_argument_count(bool &r_is_valid) const;
```

Comment:
Using reference here to avoid unnecessary null checks

Reply:
Might go back to having this be virtual, but define a default behaviour to refer back to, that way new code doesn't forget to add it